components/src/tokens/color.ts

```diff
@@ -72,10 +72,14 @@ const categories = {
 } satisfies Record<string, CategoryItem>
 
 const gradients = {
+  accent:
+    'linear-gradient(330.4deg, #44BCF0 4.54%, #7298F8 59.2%, #A099FF 148.85%)',
```

Comment:
i forgot to add the code for it, but the accent gradient should be derived in the `makeMode` function, not static

Reply:
I'll leave this to you? I don't see how it could be computed since all the gradients seem different. I also added two more gradients that were in the thorin file but not in the color.ts

Reply:
never mind... I figured out what you meant and added accent to gradients